src/internt-vedtak/BeregningsresultatContent.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { NonBreakingDiv } from './felles';
+import { Beregningsresultat } from './typer/beregningsresultat';
+
+export const BeregningsresultatContent: React.FC<{
+    beregningsresultat?: Beregningsresultat;
+}> = ({ beregningsresultat }) => {
+    if (!beregningsresultat) {
+        return null;
+    }
+    return (
+        <NonBreakingDiv>
+            <h2>Beregning</h2>
+            {beregningsresultat.tilsynBarn &&
+                TilsynBarnBeregningsresultatTabell(beregningsresultat)}
+            {beregningsresultat.læremidler &&
+                LæremidlerBeregningsresultatTabell(beregningsresultat)}
+        </NonBreakingDiv>
+    );
+};
+
+const TilsynBarnBeregningsresultatTabell = (beregningsresultat: Beregningsresultat) => (
```

Comment:
Hvis man bruker `= ({beregningsresultat}: {beregningsresultat: Beregningsresultat})` så får man brukt komponenten som en tag
`<TilsynBarnBeregningsresultatTabell bregningsresultat={beregningsresultat} />`

Burde kanskje også sende med `tilsynBarn` direkt in i denne komponenten for å unngå `?` i `beregningsresultat.tilsynBarn?`